includes/definitions/tachyon.yaml

```
@@ -9,4 +9,4 @@ over:
     - { graph: device_mempool, text: 'Memory Usage' }
 discovery:
     - sysObjectID:
-        - .1.3.6.1.4.1.57344.1
+        - .1.3.6.1.4.1.57344
```

```suggestion
        - .1.3.6.1.4.1.57344.
```

I like to leave the . so it doesn't match 573440 or something like that.  I know it is unlikely.